utils/dockerutil.py

```diff
@@ -93,16 +101,19 @@ def is_ecs(self):
 
     def get_events(self):
         self.events = []
-        should_reload_conf = False
+        conf_reload_set = set()
         now = int(time.time())
 
         event_generator = self.client.events(since=self._latest_event_collection_ts,
                                              until=now, decode=True)
         self._latest_event_collection_ts = now
         for event in event_generator:
             try:
-                if event.get('status') in CONFIG_RELOAD_STATUS:
-                    should_reload_conf = True
+                if self.config_store and event.get('status') in CONFIG_RELOAD_STATUS:
+                    image = event.get('from', '')
+                    checks = self._get_checks_from_image(image)
```

Comment:
Are we still able to identify the correct checks if the `com.datadoghq.sd.check.id` label is used on the container?